frontend/src/pages/ProgressionChecker/GridView/GridView.jsx

```diff
@@ -30,6 +36,9 @@ const GridView = ({ isLoading, structure }) => {
                 key: courseCode,
                 title: subgroupStructure.courses[courseCode],
                 completed: courseCode in pastCourses,
+                // if undefined that means course not in term planner
+                unplanned: courses[courseCode] === undefined
```

Comment:
Thinks this ternary could be simplified (albiet if we rename 'unplanned' to 'planned')

I.e.

planned: !!coures[courseCode]?.plannedFor

OR

unplanned: !courses[courseCode]?.plannedFor

Reply:
woah never seen that syntax before nice

Reply:
ended up having to use ```courses[courseCode]?.plannedFor === null``` as ```courses[courseCode]?.plannedFor``` can be undefined which is also falsey rip